tikv/mvcc.go

```diff
@@ -291,17 +293,18 @@ func (store *MVCCStore) CheckTxnStatus(reqCtx *requestCtx,
 func (store *MVCCStore) handleCheckPessimisticErr(startTS uint64, err error, isFirstLock bool) (*lockwaiter.Waiter, error) {
 	if lock, ok := err.(*ErrLocked); ok {
 		keyHash := farm.Fingerprint64(lock.Key)
+		waiter := store.lockWaiterManager.NewWaiter(startTS, lock.StartTS, farm.Fingerprint64(lock.Key), time.Second)
```

Comment:
@coocood  I moved the `NewWaiter` line in front since we need the wake-up process will send the `deadlockResponse` to the waiter's channel, any suggestions?